src/DDTrace/Configuration/EnvVariableRegistry.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace DDTrace\Configuration;
+
+/**
+ * Registry that holds configuration properties and that is able to recover configuration values from environment
+ * variables.
+ */
+class EnvVariableRegistry
```

Comment:
Would it make sense to extract interface of this?

Reply:
Absolutely, let's do it